.gitignore

```diff
@@ -0,0 +1,15 @@
+.vs/
+# Binaries
+**/bin/
+**/obj/
+# Ignore all certificate and key files
+ca.key
+ca.crt
+ca.srl
+server.key
+server.crt
+server.csr
+client.key
+client.crt
+client.csr
+passwd
```

Comment:
Lets use the standard VS gitignore as base for our gitignore. See https://github.com/github/gitignore/blob/main/VisualStudio.gitignore.

On top of that ofc we should add those certs.